content/docs/04-clusters/01-public-cloud/06-tke.md

```diff
@@ -44,6 +44,12 @@ Palette supports the deployment of tenant workloads with Tencent Kubernetes Engi
 5. A Route table set to accept external traffic, so that the nodes getting created in the associated subnets will have internet capability.
 
 
+6. Create a security group for network security isolation.
```

Comment:
This is a very vague statement, and it might be best to remove it if we don't have specific guidance to offer. Otherwise it can lead to confusion.